core/src/main/java/jenkins/util/JSONSignatureValidator.java

```diff
@@ -73,7 +75,17 @@ public FormValidation verifySignature(JSONObject o) throws IOException {
                     certs.add(c);
                 }
 
-                CertificateUtil.validatePath(certs, loadTrustAnchors(cf));
+                try {
+                    CertificateUtil.validatePath(certs, loadTrustAnchors(cf));
```

Comment:
why are we even attempting this if the certrificate is not valid (lines 70,72) above.

Isn't this the root cause the human messages that would have been displayed are swallowed as this code ran and threw an exception?